Drive/spin.hpp

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Lets put this file in its own directory rather than cluttering the drive directory with it. Maybe put it in the a `prototype/spin_testing.cpp` file with its own `main()` function call.